linode/obj/resource.go

```diff
@@ -120,6 +123,11 @@ func diffResource(
 // specified bucket via the *schema.ResourceData, then it calls
 // readResource.
 func putObject(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	// put happen before reads so add an s3 endpoint compute so it's not accidently empty
```

Comment:
```suggestion
	// put happen before reads so add an s3 endpoint compute so it's not accidentally empty
```
Suggested by the linter